linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/AdminHandler.scala

```diff
@@ -1,12 +1,11 @@
 package io.buoyant.linkerd.admin
 
-import com.twitter.finagle.http.{MediaType, Request, Response}
-import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.finagle.http.{MediaType, Response}
 import com.twitter.util.Future
 import io.buoyant.admin.HtmlView
 import io.buoyant.linkerd.Build
 
-object AdminHandler extends HtmlView {
+class AdminHandler(namerdNav: Boolean) extends HtmlView {
```

Comment:
This seems like a reasonable stopgap fix.  I've ticketed a more long term solution here: https://github.com/linkerd/linkerd/issues/972